src/blobs.jl

```diff
@@ -72,6 +72,16 @@ end
 delete(x::Container, key::String; kw...) = Azure.delete(API.makeURL(x, key); kw...)
 delete(x::Object; kw...) = delete(x.store, x.key; credentials=x.credentials, kw...)
 
+function create_append_blob(c::Container, key::String; kw...)
```

Comment:
How about we make this functionality available via `API.putObject(...; append=true)` instead of `create_append_blob` / `API.put(...; append=true)` instead of `append_block`?  It would keep similar functionality under similar API, but I'm open to alternatives, especially if there is a precedent.

Reply:
I basically mimicked the Python SDK
* see usage here: https://github.com/Azure/azure-sdk-for-python/blob/91f5a8bc5fb41273b348ee45023a715f26fb79ac/sdk/storage/azure-storage-blob/tests/test_append_blob.py#L63-L65
* docs here: https://learn.microsoft.com/en-us/python/api/azure-storage-blob/azure.storage.blob.appendblobservice.appendblobservice?view=azure-python-previous#azure-storage-blob-appendblobservice-appendblobservice-append-block

I kind of prefer the explicit names, but idk, it's a matter of taste 🤷‍♂️ The style of the repo does skew toward kwargs